src/index.js

```diff
@@ -0,0 +1,118 @@
+import './style.css';
+
+const list = document.getElementById('listtodo');
+const add = document.getElementById('add');
+
+const completed = false;
+let index = 1;
+let todoList = [];
+
+const showList = () => {
+  let litItems = '';
+  todoList.forEach((element) => {
+    if (element.completed === true) {
+      litItems += `<li class="task">
+                                  <div>
+                                  <input type="checkbox" class="checkFunc" checked onclick="checkFunc(${element.index});" id="check${element.index}">
+                                  <input type="text" value="${element.description}" class="listitem" id="item${element.index}" readonly>
+                                  </div>
+                                  <div class = "tools">
+                                  <i class="fa-solid fa-pen-to-square edit " id="edit${element.index}" onclick="editFunc(${element.index});"></i>
+                                  <i class="fa-solid fa-floppy-disk save hide box" id="save${element.index}" onclick="saveFunc(${element.index});"></i>
+                                  <i id="removeicon" onclick="removeFunc(${element.index});" class="fa-solid fa-trash"></i>
+                                  </div>
+                          </li>`;
+    } else {
+      litItems += `<li class="task">
+                                  <div>
+                                  <input type="checkbox" class="checkFunc" onclick="checkFunc(${element.index});" id="check${element.index}">
+                                  <input type="text" value="${element.description}" class="listitem" id="item${element.index}" readonly>
+                                  </div>
+                                  <div class = "tools">
+                                  <i class="fa-solid fa-pen-to-square edit " id="edit${element.index}" onclick="editFunc(${element.index});"></i>
+                                  <i class="fa-solid fa-floppy-disk save hide box" id="save${element.index}" onclick="saveFunc(${element.index});"></i>
+                                  <i id="removeicon" onclick="removeFunc(${element.index});" class="fa-solid fa-trash"></i>
+                                  </div>
+                          </li>`;
+    }
+  });
+
+  list.innerHTML = litItems;
+  add.value = '';
+};
+
+showList();
+// Add function
+add.addEventListener('keypress', (e) => {
+  if (e.key === 'Enter' && add.value.length !== 0) {
+    const storedList = localStorage.getItem('todo');
+
+    if (storedList === null) {
+      todoList = [];
+    } else {
+      todoList = JSON.parse(storedList);
+      index = todoList.length === 0 ? 0 : todoList.length;
```

Comment:
- Kindly, even if you are starting the index from `1` you using the next index to be the length which will repeat `1` two times, please use `todoList.length + 1` for your next index :+1: 

![Screenshot from 2023-02-25 18-41-41](https://user-images.githubusercontent.com/52010789/221366017-eff34734-092d-4489-988e-b85f90a921f8.png)